superset-frontend/src/components/ReportModal/index.tsx

```diff
@@ -330,7 +313,7 @@ const ReportModal: FunctionComponent<ReportProps> = ({
                 value: target.value,
               }),
           }}
-          errorMessage={currentReport?.error || ''}
+          errorMessage={renderErrorMessage() || ''}
```

Comment:
another example where this could probably be a static property/const rather than method.